llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -78,15 +78,19 @@ bool VPlanTransforms::tryToConvertVPInstructionsToVPRecipes(
         assert(!isa<PHINode>(Inst) && "phis should be handled above");
         // Create VPWidenMemoryRecipe for loads and stores.
         if (LoadInst *Load = dyn_cast<LoadInst>(Inst)) {
+          SmallVector<std::pair<unsigned, MDNode *>> Metadata;
+          ::getMetadataToPropagate(Inst, Metadata);
```

Comment:
```suggestion
          VPIRMetadata Metadata(Inst);
```
?

Reply:
Done thanks

Reply:
Still awaits commit? Defining Metadata here is now redundant.